src/allowed-blocks-setup.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * External dependencies
+ */
+import { map, without } from 'lodash';
+/**
+ * WordPress dependencies
+ */
+import { dispatch } from '@wordpress/data';
+import { getBlockTypes } from '@wordpress/blocks';
+/**
+ * Internal dependencies
+ */
+import { JETPACK_DATA_PATH } from '../jetpack/projects/plugins/jetpack/extensions/shared/get-jetpack-data';
+import { registerBlock as registerJetpackLayoutGridBlock } from '../block-experiments/blocks/layout-grid/src';
+import {
+	registerContactInfoBlock as registerJetpackContactInfoBlock,
+	registerStoryBlock as registerJetpackStoryBlock,
+} from '../jetpack/projects/plugins/jetpack/extensions/editor.native';
+
+// Please also consider updating ./block-support/supported-blocks.json
+const availableJetpackBlocks = {
+	'contact-info': { available: true },
+	'layout-grid': { available: true },
+	story: { available: true },
+};
+
+const setJetpackData = ( {
```

Comment:
That crossed my mind at one point as well and I simply forgot. Thanks for reminding me. I couldn't think of a good name at the time though. Naming is hard.

`toJetpackData`, `mapToJetpackData`, or similar aren't really that much more expressive even if they don't read like an accessor method name. WDYT?

- [ ] Rename the function to something more accurate.

Reply:
Yep, naming is hard 😄 , to be honest, I don't have a better name in mind but I'd probably avoid using `set` in the name as it could be a bit misleading. Being this said, `mapToJetpackData` looks slightly better so I think it's a good alternative.

Reply:
Done.